django/api/settings.py

```diff
@@ -45,6 +45,7 @@
     'api.apps.ApiConfig',
     'corsheaders',
     'django_filters',
+    'django_extensions',
```

Comment:
These are already installed as an app.

Reply:
Ah nice catch @naomiaro :)

Reply:
This duplication hasn't been removed yet.

Reply:
Right... Cause I just removed it from [fetch.py](https://github.com/popkinj/itvr/blob/release-0.1.0/django/api/jobs/hourly/fetch.py). Didn't realize there was a duplicate in settings.py as well. Will take care of it.